shell.nix

```
@@ -81,6 +87,7 @@ let
       tmux
       pkgs.git
       pkgs.hlint
+      pkgs.glibcLocales
     ]
```

```suggestion
    ] ++ lib.optional haveGlibcLocales pkgs.glibcLocales
```
I forgot to mention that macOS and musl libc targets don't have this package, so it needs to be conditional.